packages/twenty-front/src/modules/object-record/record-group/states/recordGroupDefinitionFamilyState.ts

```diff
@@ -0,0 +1,10 @@
+import { RecordGroupDefinition } from '@/object-record/record-group/types/RecordGroupDefinition';
+import { atomFamily } from 'recoil';
+
+export const recordGroupDefinitionFamilyState = atomFamily<
```

Comment:
what is the recordGroupDefinition used for? I feel it's not useful?

Reply:
RecordGroupDefinition is almost a mapping of view-groups, at first it was added because of the NoValue group that is not stored in the backend side. But as we now wan't it to act as a normal RecordGroup this one is stored in the backend too.
But I think we should keep it now for consistency in the code.